bedrock/firefox/templates/firefox/family/includes/modules/private-mode.html

```diff
@@ -24,7 +24,7 @@
           </defs>
         </svg>
         <h3 class="c-subtitle">The private browsing mode paradox</h3>
-        <p>On the one hand it keeps your kid’s data from being collected
+        <p>On the one hand, private browsing can make it harder for advertisers to track your kid
```

Comment:
Suggest making kid plural?

```suggestion
        <p>On the one hand, private browsing can make it harder for advertisers to track your kids
```

Reply:
I think it's meant to be singular, most of the other sections use "your kid" too